src/queryResult/utils.ts

```diff
@@ -217,6 +218,26 @@ export function registerCommonRequestHandlers(
             return state;
         },
     );
+    webviewController.registerReducer(
+        "setFilterState",
+        async (state, payload) => {
+            if (!state.filterState[payload.filterState.columnDef]) {
```

Comment:
what does this if check cover? Wouldn't the part below this (Line232 - 237) already work if `state.filterState` doesn't have `columnDef` property?